src/Commands/ComponentCommand.php

```diff
@@ -15,7 +15,7 @@ public function handle()
     {
         $name = Str::studly($this->argument('name'));
 
-        if (! is_dir($path = app_path('Airwire'))) {
+        if (!is_dir($path = app_path('Airwire'))) {
```

Comment:
Revert the style change

```suggestion
        if (! is_dir($path = app_path('Airwire'))) {
```